app/src/main/java/com/eveningoutpost/dexdrip/utils/Preferences.java

```diff
@@ -470,20 +471,25 @@ public void onClick(DialogInterface dialog, int which) {
             });
         }
 
-        private static Preference.OnPreferenceChangeListener sBgMissedAlertsHandler = new Preference.OnPreferenceChangeListener() {
+        private static Preference.OnPreferenceChangeListener sAlertsPreferanceHandler = new Preference.OnPreferenceChangeListener() {
             @Override
             public boolean onPreferenceChange(Preference preference, Object newValue) {
+            	// some of the alerts parameters have changed, let's make sure that we don't need to alert.
                 Context context = preference.getContext();
                 context.startService(new Intent(context, MissedReadingService.class));
+                context.startService(new Intent(context, Notifications.class));
                 return true;
             }
         };
 
         
         private void bindBgMissedAlertsListener(){
-          findPreference("bg_missed_alerts").setOnPreferenceChangeListener(sBgMissedAlertsHandler);
-          findPreference("bg_missed_minutes").setOnPreferenceChangeListener(sBgMissedAlertsHandler);
-          findPreference("other_alerts_snooze").setOnPreferenceChangeListener(sBgMissedAlertsHandler);
+          findPreference("bg_missed_alerts").setOnPreferenceChangeListener(sAlertsPreferanceHandler);
+          findPreference("bg_missed_minutes").setOnPreferenceChangeListener(sAlertsPreferanceHandler);
+          findPreference("other_alerts_snooze").setOnPreferenceChangeListener(sAlertsPreferanceHandler);
+          findPreference("alerts_disabled_until").setOnPreferenceChangeListener(sAlertsPreferanceHandler);
```

Comment:
This "setting" isn't changed by the user and not in this activity.
It will be changed programmatically after a button press in the SnoozeActivity.